Apps/Inc/ReadCarCAN.h

```diff
@@ -11,10 +11,28 @@
 
 #define SAT_BUF_LENGTH 5
 
+#define __TEST_READCARCAN
+
+#ifdef __TEST_READCARCAN
+extern int chargeMsgSaturation;
+extern uint8_t SOC;
+extern uint32_t SBPV; 
+#endif
+
+/**
+ * Error types
+ */
+typedef enum{
+	READCARCAN_ERR_NONE			    = 0,
+	READCARCAN_ERR_CHARGE_DISABLE   =-1,	// Received a charge disable msg
+	READCARCAN_ERR_MISSED_MSG		=-2,	// Didn't receive a BPS charge msg in time
+	READCARCAN_ERR_BPS_TRIP		    =-3		// Received a BPS trip msg (0 or 1)
```

Comment:
Just a stylistic change. You probably just want the enum to assign the default values, instead of incrementing them by their negative numbers. Error values don't have to be negative numbers. Instead, just let the NONE error code be default (which is at, 0) and let the others be 1, 2, ... and so on. This way, you don't have to keep assigning values if you add more errors

Reply:
@KnockbackNemo Could we do this?

Reply:
@diyarajon Yup, that sounds good